Sources/ArcGISToolkit/Components/UtilityNetworkTrace/UtilityNetworkTraceViewModelTrace.swift

```diff
@@ -82,19 +82,21 @@ extension UtilityNetworkTraceViewModel {
         
         /// The extent of the trace's geometry result with a small added buffer.
         var resultExtent: Envelope? {
-            if let resultEnvelope = GeometryEngine.combineExtents(of: [
+            let geometries = [
                 utilityGeometryTraceResult?.multipoint,
                 utilityGeometryTraceResult?.polygon,
                 utilityGeometryTraceResult?.polyline
-            ].compactMap { $0 }),
-               let expandedEnvelope = GeometryEngine.buffer(
-                around: resultEnvelope,
-                distance: 200
-               ) {
-                return expandedEnvelope.extent
-            } else {
+            ]
+                .compactMap { $0 }
+                .filter { !$0.isEmpty }
```

Comment:
Not a big deal in this case because the collection is small, but in general, it is best to iterate the collection as few times as possible. In this case, the filtering could be done in the transform closure when compact mapping.

```suggestion
                .compactMap { geometry in
                    if let geometry, !geometry.isEmpty {
                        return geometry
                    } else {
                        return nil
                    }
                }
```